### PR TITLE
Installer cleanup

### PR DESCRIPTION
**Pull request by codebrain**
- General cleanup of installer code
- Launch correct ODBC control panel after installation. Implemented as a CustomAction as using the WixShellExecTarget only ever launches a 32 bit process, then attempting to access any 64 bit programs will succumb to windows path rewriting and resolve to 32 bit equivalent program.

### CODE REVIEW

File: installer/src/Installer/Program.cs [hunk 1]
```diff
@@ -209,9 +199,30 @@ private static FileVersionInfo GetDriverFileInfo(string zipContentsDirectory)
 		}
 	}
 
+	// Implemented as a CustomAction as using the WixShellExecTarget only ever launches a 32 bit process,
+	// then attempting to access any 64 bit programs will succumb to windows path rewriting and resolve to
+	// 32 bit equivalent program.
+	public class CustomActions
```
Comment (russcam) [inline installer/src/Installer/Program.cs]: I think this custom action should be implemented in a separate DLL, and included in the installer files. As is, the complete Installer program exe will be bundled into the installer just for this one custom action.

Comment (codebrain) [reply installer/src/Installer/Program.cs]: Fair point, will extract and reference.

Comment (codebrain) [reply installer/src/Installer/Program.cs]: @russcam - Changes made as requested.

File: installer/src/InstallerCA/CustomActions.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+{
+	// Implemented as a CustomAction as using the WixShellExecTarget only ever launches a 32 bit process,
+	// then attempting to access any 64 bit programs will succumb to windows path rewriting and resolve to
+	// 32 bit equivalent program.
+	public class CustomActions
+	{
+		[CustomAction]
+		public static ActionResult LaunchODBCControlPanel(Session session)
+		{
+			var windir = Environment.GetEnvironmentVariable("windir");
```
[the remark this answers is not in the document]
Comment (codebrain) [reply installer/src/InstallerCA/CustomActions.cs]: ++, will change and merge

**Pull request merged**